src/main/java/step4/RacingCarMain.java

```diff
@@ -0,0 +1,17 @@
+package step4;
+
+import step4.controller.RacingCarController;
+import step4.view.InputView;
+import step4.view.OutputView;
+
+public class RacingCarMain {
+    public static void main(String[] args) {
+        final String carNames=InputView.getCarNames();
+        final int tryNo=InputView.getRaceCount();
```

Comment:
변수에 final은 여기에만 선언 되어 있는 것 같아요.
일관성을 저해하는데 제거하면 어떨까요?

\+ 띄어쓰기도 신경써주세요!
 ```suggestion
        String carNames = InputView.getCarNames();
        int tryNo = InputView.getRaceCount();
```

Reply:
아직 final과 static을 정확히 언제 선언하면 좋을지 명확하게 알지 못해 그랬던 것 같습니다. 다음에는 기능과 일관성을 모두 신경써서 사용하도록 하겠습니다 감사합니다!🙇‍♂️